### PR TITLE
Use new ink entrance crate

### PR DESCRIPTION
**Pull request by ascjones**
Companion for https://github.com/paritytech/ink/pull/1223

### CODE REVIEW

**Review by cmichi — APPROVED** [review]
May I ask for a `CHANGELOG.md` entry 😬 ?

File: crates/cargo-contract/templates/new/_Cargo.toml [hunk 1]
```diff
@@ -5,11 +5,8 @@ authors = ["[your_name] <[your_email]>"]
 edition = "2021"
 
 [dependencies]
-ink_primitives = { version = "4.0.0-alpha.1", default-features = false }
-ink_metadata = { version = "4.0.0-alpha.1", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "4.0.0-alpha.1", default-features = false }
-ink_storage = { version = "4.0.0-alpha.1", default-features = false }
-ink_lang = { version = "4.0.0-alpha.1", default-features = false }
+# todo: update to crates.io version once released
```
Comment (cmichi) [inline crates/cargo-contract/templates/new/_Cargo.toml:8]: We could do a `4.0.0-alpha.2`?

Comment (ascjones) [reply crates/cargo-contract/templates/new/_Cargo.toml:8]: I have a follow up to do but yes ASAP

**Review by sergejparity — APPROVED** [review]

File: crates/cargo-contract/templates/new/_Cargo.toml (hunk 1 above)
Comment (ascjones) [reply crates/cargo-contract/templates/new/_Cargo.toml:8]: https://github.com/paritytech/ink/pull/1400

**Pull request merged**